src/css/_variables.scss

```diff
@@ -0,0 +1,10 @@
+//Variables
+
+//Colors
+$primary-background-color: #EBF5EE;
+$button-hover-color: #cff2da;
+$button-active-color: #C3C2C6;
+
+//Borders
+$box-border: 1px solid #000;
+$main-border: 2px solid #000;
```

Comment:
I think if we want to incorporate more variables we could refactor the color scheme to be more extensive